code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -1,5 +1,21 @@
 #define DEFAULT_DOOMSDAY_TIMER 4500
 
+GLOBAL_LIST_INIT(blacklisted_malf_machines, typecacheof(list(
+		/obj/machinery/field/containment,
+		/obj/machinery/power/supermatter_shard,
+		/obj/machinery/dominator,
+		/obj/machinery/doomsday_device,
+		/obj/machinery/nuclearbomb,
+		/obj/machinery/nuclearbomb/selfdestruct,
+		/obj/machinery/nuclearbomb/syndicate,
+		/obj/machinery/syndicatebomb,
```

Comment:
So why are you removing it from malf control if even normal AIs can set the timer? That makes even less sense and makes me think this is a nerf rather than a bugfix.